tests/transport_test.go

```diff
@@ -0,0 +1,118 @@
+package tests
+
+import (
+	"fmt"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/pkg/errors"
+	"kubevirt.io/containerized-data-importer/pkg/common"
+	"kubevirt.io/containerized-data-importer/tests/framework"
+	"kubevirt.io/containerized-data-importer/tests/utils"
+
+	"k8s.io/api/core/v1"
+)
+
+var _ = Describe("Transport Tests", func() {
+
+	const (
+		annEndpoint = "cdi.kubevirt.io/storage.import.endpoint"
+		annSecret   = "cdi.kubevirt.io/storage.import.secretName"
```

Comment:
If these strings already defined in the _controller_ pkg cannot be imported then maybe they should be moved to _common_? I'd prefer to no see these strings hard-coded in more than 1 place.

Reply:
Ah!  I didn't see them in common so I wrote them in here. Definitely better to import from pkg/controller